Rubberduck.Parsing/ComReflection/ComMember.cs

```diff
@@ -36,10 +36,13 @@ public class ComMember : ComBase
         public bool IsEnumerator { get; }
         //This member is called on an interface when a bracketed expression is dereferenced.
         public bool IsEvaluateFunction { get; }
-        public ComParameter ReturnType { get; private set; } = ComParameter.Void;
+        public ComParameter AsTypeName { get; private set; } = ComParameter.Void;
 
-        private readonly List<ComParameter> _parameters = new List<ComParameter>();
-        public IEnumerable<ComParameter> Parameters => _parameters;
+        private List<ComParameter> _parameters = new List<ComParameter>();
+
+        //See https://docs.microsoft.com/en-us/windows/desktop/midl/retval
+        //"Parameters with the [retval] attribute are not displayed in user-oriented browsers."
+        public IEnumerable<ComParameter> Parameters => _parameters.Where(param => !param.IsReturnValue);
```

Comment:
Do we have a use cases where we need to know about the parameters from the type lib's POV, rather than from user's POV?

Reply:
None currently that I can think of off the top of my head, although it would make generating MIDL easier if we decide to do that in the future.